test/https.ts

```diff
@@ -421,7 +421,7 @@ test('invalid key passphrase', withHttpsServer(), async (t, server, got) => {
 			await request;
 			t.fail();
 		} catch (error) {
-			t.true((error.message as string).includes('bad decrypt'));
+			t.regex(error.message, /bad decrypt/);
```

Comment:
The `.includes` was was better. No point in using regex if it's not necessary.

Reply:
Using the `regex(a, b)` you can see the message if regex doesn't match, with `true(a.includes(b))` you only see something like "expected 'true' found 'false'".

That's the reason why I changed it (take a look at this: https://travis-ci.com/github/sindresorhus/got/jobs/384356165)

Reply:
You can use the second argument to see the actual value.

```js
t.true((error.message as string).includes('bad decrypt'), error.message);
```